packages/aws-cdk-lib/aws-eks/lib/managed-nodegroup.ts

```diff
@@ -426,7 +430,7 @@ export class Nodegroup extends Resource implements INodegroup {
        * 1. instance types of different CPU architectures are not mixed(e.g. X86 with ARM).
        * 2. user-specified amiType should be included in `possibleAmiTypes`.
        */
-      possibleAmiTypes = getPossibleAmiTypes(instanceTypes);
+      possibleAmiTypes = getPossibleAmiTypes(instanceTypes).concat(NodegroupAmiType.CUSTOM);
```

Comment:
Can you please concatenate inside the method for reusability?

Reply:
Moved :)